TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/factories.py

```diff
@@ -67,7 +67,13 @@ def get_replayer(config: Dict):
 def get_kafka(config: Dict):
     if 'msk' in config:
         return MSK(config)
-    return StandardKafka(config)
+    if 'standard' in config:
+        return StandardKafka(config)
+    config.pop("broker_endpoints", None)
+    logger.error(f"An unsupported kafka source type was provided: {config.keys()}")
+    if len(config.keys()) > 1:
+        raise UnsupportedKafkaError(', '.join(config.keys()))
```

Comment:
all of the cases should be covered by this join, "" if empty, "item" if only has one value so line 74 and 76 should be able to be removed

Reply:
Oh, good catch. thanks, fixing now